### DISCUSSION TITLE
蓝灯不给退款的朋友看进来，大家一起搞

### DISCUSSION THREAD

**Discussion by David68326** [getlantern/lantern-forum-cn#804] [General]
蓝灯不能正常使用，售后又很垃圾，不能用还不能退款，看看论坛里有多少申请退款被无赖拖延的我们可以一起投诉，团结一致，人多力量大，争取退款。有热心朋友已经准备好了投诉模板。
支付宝投诉链接 https://110.alipay.com/sc/report.htm
购买了蓝灯加速器后发现无法使用，商家拒绝退款，涉嫌欺诈。商家stripe支付宝账号为Wellnity,LLC，另有其他账号： stripe支付宝公司账号 bytesteachrecruiting 退款帖链接：https://github.com/getlantern/lantern-forum-cn/discussion/574 stripe支付宝公司账号 nevernetworks.com 退款帖链接：https://github.com/getlantern/lantern-forum-cn/discussions/560 stripe支付宝公司账号 newladders 退款帖链接：https://github.com/getlantern/lantern-forum-cn/discussions/610 stripe支付宝公司账号 fairstride 退款帖链接：https://github.com/getlantern/lantern-forum-cn/discussions/683 请求支付宝退款并封禁这些不良商家账号。

Stripe投诉连接 https://stripe.com/zh-us/complaints 投诉模板也已经翻译成英文
After purchasing the Lantern VPN with alipay via stripe, I found it unusable, and the merchant refused a refund, suspected of fraud. The merchant's Stripe Alipay account is Wellnity, LLC, with additional accounts: Stripe Alipay company account bytesteachrecruiting refund complaint post link: https://github.com/getlantern/lantern-forum-cn/discussion/574 Stripe Alipay company account nevernetworks complaint post link: https://github.com/getlantern/lantern-forum-cn/discussions/560 Stripe Alipay company account newladders complaint post link: https://github.com/getlantern/lantern-forum-cn/discussions/610 and Stripe Alipay company account fairstride complaint post link: https://github.com/getlantern/lantern-forum-cn/discussions/683 I request stripe to process a refund and block these dishonest merchant accounts.

**Comment by David68326** [comment]
有想退款的朋友可以根据你付款时候的收款方进行投诉，除了Wellnity, LLC,还有好几个，把知道的收款公司名都附上最好，让他们不能再变换收款公司名称行骗。另外还可以去Play商店给蓝灯差评。

**Reply by JMaster1937** [reply]
好的，会继续维权到底

**Reply by MaxMKMax** [reply]
@JMaster1937 我在您的帖子都说给您退款了. 但需要您提供账号邮箱和支付单据啊.

**Comment by MaxMKMax** [comment]
@David68326 大哥,我们都说你可以在支付宝那里直接提交退款. 最好的方法就是你直接在支付宝里撤销付款. 我们这里系统已经过期了, 不允许我们退款. 您早点申请退款我们是能退的. 不是我们不推, 是完全没办法退. 国外的钱不是随便就能打到国内的. 您懂的都懂! 

我们什么时候没给人退款? 只要一个月内的申请退款我们全部都退.

同时我这里也帮你想办法了. 我等待回复. 我再给你知道!

**Reply by David68326** [reply]
@Mike777Khoo 从4月份开始我就要求退款了，开始说3、4个月的可以退，后来看我是3个月又改口说2个月内的才能退，现在又变成1个月了，一直这么拖着到现在快半年了吧？你可以去查查都说过什么。现在的问题不是几个月不几个月，是蓝灯无法正常使用，我办过几次两年的会员了，从没要求过退款，我是觉得就算有点问题基本上能正常使用就可以接受，但现在不同了，不但正常使用不了，连会员身份都不见了，经常是免费用户状态，提出过多少次，你们都解决不了，我要求退款不是很正常吗？

**Comment by JMaster1937** [comment]
支付宝支付单号为：2024080622001303891444899483 ***@***.*** 如国内限制原因可将我的会员转到
***@***.*** 的账号上 请尽快解决！

Mike K ***@***.***> 于2024年8月24日周六 02:07写道：

**Comment by JMaster1937** [comment]
***@***.***

Jack Ma ***@***.***> 于2024年8月24日周六 19:30写道：

**Comment by heidelbergkai** [comment]
气死了，不给解决。

**Comment by heidelbergkai** [comment]
一个月个屁啊，你们不能用多久了，你们自己不知道吗？不能用的时间也要加进去？

**Comment by greySkiesMarsh** [comment]
@David68326 

Hello, 

My name is Marsh, and I'm the Customer Support Lead here at Lantern. 

As stated previously by our support department, it is impossible to make refunds or changes to payments this long after the initial processing date. Additionally, our technical support team has sent messages on this matter that went unanswered. 

Unfortunately, service interruptions do sometimes occur due to the changing tactics of censors, but our team is working on updating our protocols to increase reliability. Please do communicate with our support team by submitting an in-app ticket, and we’ll do everything we can to help improve the Lantern quality of service.

But a refund is not possible, and we need to allow this forum to be focused on identifying service interruptions and improving Lantern reliability for everybody.

Thank you, 

Marsh
Lantern Support Lead
